starter-code/css/main.css

```diff
@@ -6,3 +6,157 @@ Green: #00B172
 White: #FFF
 
 */
+body {
+  font-family: 'Montserrat', sans-serif;
+  color: #1A1A1A;
+  margin: 0;
+}
+
+.navbar {
+  background-color: #fff;
+  position: fixed;
+  height: 60px;
+  padding: 10px 10px 0 0;
+  width: 100%;
+  top: 0;
+  z-index: 1;
+}
+
+.nav-ul {
+  list-style-type: none;
+  float: right;
+  margin: 0 30px 0 0;
+  padding-top: 15px;
+  font-size: 1em;
+  font-weight: 300;
+}
+
+.nav-ul > li {
+      display: inline;
+      margin: 10px;
+}
+
+.spotify-logo {
+  max-height: 80%;
+  float: left;
+  margin: 0 0 10px 0;
+  padding-left: 30px;
+}
+
+.clearfloats {
+  clear: both;
+}
+
+.background-image {
+  background-image: url("../images/landing.jpg");
+  background-size:cover;
+  background-repeat: no-repeat;
+  height: 800px;
+}
+
+.tagline {
+  margin: 0px 0 20px 0;
+  padding: 200px 0 0 0;
+  font-size: 4em;
+  text-align: center;
+  color: #fff;
+}
+
+.subtext {
+  font-size: 1.2em;
+  text-align: center;
+  margin: 0 0 0 0;
+  color: #fff;
+  font-weight: 200;
+}
+
+.features {
+  text-align: center;
+}
+
+.features h3 {
+  padding-bottom: 4px;
+  font-size: 1.8em;
+  display: inline-block;
+  border-bottom: 3px solid #00B172;
+
+}
+
+.features-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.features div {
+  text-align: center;
+  margin: 30px 30px;
+}
+
+.features div img {
+  max-width: 40%;
+}
+
+.features div h4 {
+  color: #00B172;
+  font-size: 1.2em;
+  font-weight: bold;
+}
+
+.yeezy-green {
+  background-color: #00B172;
+  margin: 30px;
+  position: relative;
+}
+
+.yeezy-phone {
+  position: absolute;
```

Comment:
Don't use `position: absolute`if you can avoid it. Here, flexbox or floats can do the job.